hrjobcontract/js/test/services/contract-pay_test.js

```diff
@@ -16,8 +16,8 @@ define([
       $httpBackend = _$httpBackend_;
       $rootScope = _$rootScope_;
 
-      $httpBackend.whenGET(/action=get&entity=HRJobContract/).respond({});
-      $httpBackend.whenGET(/action=get&entity=HRJobPay/).respond(MockContract.contractPayment);
+      $httpBackend.whenGET(/action=get&entity=HRJobContract/).respond(MockContract.contractPayment);
```

Comment:
@AkA84 explained to me that it was an error introduced by copy pasting the tests. Since tests were "passing" the mistake was not noticed.